### ISSUE TITLE
auto-playlist for all music

### ISSUE THREAD

**Issue by Oleksiy-Yakovenko** [DeaDBeeF-Player/deadbeef#838]
Original [issue 936](https://code.google.com/p/ddb/issues/detail?id=936) created by Alexey-Yakovenko on 2013-01-29T03:50:43.000Z:

Description:

Foobar2000 has a way to add your library of music to an internal database, and then you can create an always-updated playlist of all the music in that database. It would be wonderful if deadbeef had some analogue to this. 

Also useful would be just general auto-updating playlists based on search terms or so, foobar2000 also does these. I think this is the more-generic implementation of my specific request. 

Thank you :)

**Comment by mdooligan** [comment]
I implemented this type of thing once by allowing a 'listCreateCommand' key in the playlist db, eg:

[code]
listCreateCommand=find /mnt/Audio -follow -iregex ".*Vivaldi.*.\(mp3\|flac\)"
[/code]
and having a menu item File -> Re-create playlist...'
which pops up a dialog showing the command to edit, and just hit ENTER to execute using popen() or even system().

**Comment by shiro** [comment]
Any progress on this or suggestions how this would be implemented?
Thanks for the amazing work on this project!

**Comment by Alexey-Yakovenko** [comment]
Deadbeef doesn't have internal database with all music, so this can't happen until the database is implemented.

**Comment by 4oo4** [comment]
@Alexey-Yakovenko Is there an issue tracker for the database implementation?

**Comment by Alexey-Yakovenko** [comment]
@4oo4 deadbeef only has 1 issue tracker, you're writing into it.

**Comment by 4oo4** [comment]
@Alexey-Yakovenko yes, I know that. I meant to say is there a specific issue for the database implementation that I can follow?

**Comment by Alexey-Yakovenko** [comment]
We use slack to discuss development.
